### PR TITLE
[Feature] Support show tablets stmt

### LINKED ISSUE #7969: [Feature] Support show tablets stmt
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.

### Description

query `show tablets` after querying `show partitions`, its not supported. 

### Use case

_No response_

### Related issues

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by ChPi**
# Proposed changes

Issue Number: close #7969 

## Problem Summary:

support show `tablets stmt`.

## Checklist(Required)

1. Does it affect the original behavior: (No)
2. Has unit tests been added: (No Need)
3. Has document been added or modified: (No)
4. Does it need to update dependencies: (No)
5. Are there any changes that cannot be rolled back: (No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by EmmyMiao87** [comment]
It seems that the current ```show tablet``` can completely cover the needs of ```show tablets```. 
Is it unnecessary to add this api?

**Comment by yangzhg** [comment]
> It seems that the current `show tablet` can completely cover the needs of `show tablets`. Is it unnecessary to add this api?

like show tables， may be show tablets is more reasonable

**Review by yangzhg — DISMISSED** [review]
LGTM

**Comment by caiconghui** [comment]
I think it show tablets + id seems confused, also not a good way to supply two ways to get tablets from table @EmmyMiao87 @yangzhg

**Comment by yangzhg** [comment]
> tablets

yes it should be. `show tablet id` and `show tablets from table`

**Comment by yangzhg** [comment]
> I think it show tablets + id seems confused, also not a good way to supply two ways to get tablets from table @EmmyMiao87 @yangzhg
Yes! It should be `show tablet id` and `show tablets from table`

**Comment by ChPi** [comment]
> I think it show tablets + id seems confused, also not a good way to supply two ways to get tablets from table @EmmyMiao87 @yangzhg

yes，i delete show tablets + id .

only support `show tablets from table` like `show partitions from table`.

**Comment by caiconghui** [comment]
please alse update doc, please @ChPi

**Comment by ChPi** [comment]
> please alse update doc @ChPi

docs are copied from `show tablet` and made some changes, please take a look

File: docs/zh-CN/sql-reference/sql-statements/Data Manipulation/SHOW TABLETS.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# SHOW TABLET
```
Comment (caiconghui) [inline docs/zh-CN/sql-reference/sql-statements/Data Manipulation/SHOW TABLETS.md:27]: show tablets

**Comment by caiconghui** [comment]
and you also need modify show tablet doc, since that we only support show tablets and show tablet id now

File: fe/fe-core/src/main/cup/sql_parser.cup [hunk 2]
```diff
@@ -2743,6 +2743,10 @@ show_param ::=
     {:
```
Comment (caiconghui) [inline fe/fe-core/src/main/cup/sql_parser.cup:2743]: remove show tablet from

Comment (ChPi) [reply fe/fe-core/src/main/cup/sql_parser.cup:2743]: 我觉得为了兼容性应该暂时先保留着，万一有使用者在自己系统用了这个语法，升级后他们系统就会出错

Comment (caiconghui) [reply fe/fe-core/src/main/cup/sql_parser.cup:2743]: actually, if for compatibility, we will always cannot remove the old grammar,  or show just close this modification? I think it is not good to support two grammar  to do same thing forever @EmmyMiao87 @yangzhg what't your suggestion?

Comment (morningman) [reply fe/fe-core/src/main/cup/sql_parser.cup:2743]: +1 for @caiconghui , we'd better remove the old grammer, and this should be noticed in CHANGELOG

Comment (ChPi) [reply fe/fe-core/src/main/cup/sql_parser.cup:2743]: i will remove it

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**